packages/ts/file-router/src/shared/internal.d.ts

```diff
@@ -11,13 +10,8 @@ export type ServerViewConfig = Readonly<{
 }> &
   ViewConfig;
 
-export type ServerViewMapItem = Readonly<{
-  params?: Readonly<Record<string, RouteParamType>>;
-}> &
-  ViewConfig;
-
 export type VaadinObject = Readonly<{
-  views: Readonly<Record<string, ServerViewMapItem>>;
+  views: Readonly<Record<string, ViewConfig>>;
```

Comment:
I still wonder whether the views with optional parameters should be available as menus.